reporter/src/main/kotlin/HowToFixTextProviderScriptTemplate.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.reporter
+
+import kotlin.script.experimental.annotations.KotlinScript
+import kotlin.script.experimental.api.ScriptCompilationConfiguration
+import kotlin.script.experimental.api.defaultImports
+
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.utils.scripting.OrtScriptCompilationConfiguration
+
+@KotlinScript(
+    displayName = "ORT How-To-Fix-Text-Provider Script",
+    fileExtension = "how-to-fix-text-provider.kts",
+    compilationConfiguration = HowToFixTextProviderScriptCompilationConfiguration::class
+)
+open class HowToFixTextProviderScriptTemplate(
+    val ortResult: OrtResult
+)
+
+class HowToFixTextProviderScriptCompilationConfiguration : ScriptCompilationConfiguration(
+    OrtScriptCompilationConfiguration(),
+    body = {
+        defaultImports(
+            "org.ossreviewtoolkit.model.*",
+            "org.ossreviewtoolkit.model.config.*",
+            "org.ossreviewtoolkit.model.licenses.*",
+            "org.ossreviewtoolkit.model.utils.*",
+            "org.ossreviewtoolkit.reporter.HowToFixTextProvider"
+        )
```

Comment:
Because they are already part of

https://github.com/oss-review-toolkit/ort/blob/3bb07089b2bcd3313a91276fc71628ed7e5fa059/utils/scripting/src/main/kotlin/OrtScriptCompilationConfiguration.kt#L50-L54

and `defaultImports` accumulate.